src/components/TodoItem.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC } from 'react';
+import { BsPencilSquare, BsFillTrashFill } from 'react-icons/bs';
+
+import type Todo from '../models/todo';
+
+// props type
+export type HandleToggleCompleted = (todo: Todo) => void;
+export type HandleClickEdit = (todoId: string) => void;
+export type HandleClickRemove = (todoId: string) => void;
+
+type Props = {
+  todo: Todo;
+  handleToggleCompleted: HandleToggleCompleted;
+  handleClickEdit: HandleClickEdit;
+  handleClickRemove: HandleClickRemove;
+};
+
+const TodoItem: FC<Props> = (props) => {
```

Comment:
好み？

デフォルト値の割り当てやすさを考慮して↓みたいな書き方のほうがよく見る気がするけど、言うほど色んなコード読んだ訳でもないのが間違っているかも。

```suggestion
const TodoItem: FC<Props> = ({ todo, handleToggleCompleted, dummy = 'hoge' }) => {
```

Reply:
なるほど！

Reply:
修正しました。